src/main/java/com/nerodesk/om/aws/CdShortUrl.java

```diff
@@ -50,6 +52,10 @@
 @EqualsAndHashCode
```

Comment:
@dmzaytsev I think that the only thing you need for this to work is `@EqualsAndHashCode(of = 'decorated')`